_posts/2023-02-16-coprs-migration-to-github.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Copr's migration to GitHub
+author: Jiří Kyjovský
+layout: post
+---
+
+For a very long time, Copr mainliners have been considering migrating from
+Pagure to GitHub. The problem has been the constant problems with Pagure
+and even though GitHub is not opensource, it just works unlike Pagure. Not
+to mention that GitHub is miles ahead of Pagure with plenty of useful
+features.
+
+So on the migration itself, Copr had a mirror on GitHub at the time, so
+the code migration wasn't a problem. The problem was migrating all the issues
+and pull requests so that their IDs were preserved. We wanted to keep the
+same IDs for compatibility with the links in the commits. And there
+is no general solution to migrate all issues and PRs from Pagure to GitHub
+while preserving the IDs. So that's why I wrote a temporary
+[script](https://github.com/nikromen/copr-gh-migration/blob/main/script.py)
+to migrate Copr. Please note that the code was only meant to be used once
+and only for the Copr migration, so it's pretty much a piece of garbage
```

Comment:
I would go with

only for the Copr migration, so it ~~is pretty much a piece of garbage~~ should not be used by anyone.

I find it funnier